### PR TITLE
JBPM-7685: disabling pooling from Narayana

### PR DESCRIPTION
**Pull request by mareknovotny**
this a tryout to disable Narayana database connection pooling for tests.

related to 
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/804
* https://github.com/kiegroup/droolsjbpm-integration/pull/1554**

### CODE REVIEW

File: kie-test-util/src/main/java/org/kie/test/util/db/PoolingDataSource.java [hunk 1]
```diff
@@ -126,12 +126,12 @@ public Connection getConnection() throws SQLException {
     @Override
     public Connection getConnection(String username, String password) throws SQLException {
         Properties properties = new Properties();
+        properties.put(TransactionalDriver.poolConnections, false);
```
Comment (rsynek) [inline kie-test-util/src/main/java/org/kie/test/util/db/PoolingDataSource.java]: The `Properties` class is a bit misused in the `TransactionalDriver`, but this particular property is accessed as a property - it's a `String`, not `Boolean`. Please see https://github.com/jbosstm/narayana/blob/5.9.0.Final/ArjunaJTA/jdbc/classes/com/arjuna/ats/internal/jdbc/ConnectionManager.java#L61.

**Review by rsynek — CHANGES_REQUESTED** [review]

**Comment by mareknovotny** [comment]
Jenkins retest this

**Comment by mareknovotny** [comment]
@krisv @mariofusco @baldimir can one of you look at the issues while dbcp is disabled?

**Comment by baldimir** [comment]
@rsynek tagging you so you get notified about everything Narayana related.

**Comment by mareknovotny** [comment]
ok, one idea showed up, so my best effort ends here to change `persistence.xml` too

**Comment by mareknovotny** [comment]
So this test results correspond with FDB in https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/804

**Comment by mareknovotny** [comment]
so @rsynek did update the persistence test code, so this should not be needed.

**Pull request closed without merging**